### PR TITLE
User can rotate pedals

### PR DESCRIPTION
**Pull request by ArthurHwang**

### CODE REVIEW

File: client/components/BuilderPedals/BuilderPedals.js [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+import { Image, Icon } from 'semantic-ui-react'
 import Draggable from 'react-draggable'
 
-const BuilderPedals = ({ pedals }) => {
+const BuilderPedals = props => {
   return (
-    <div style={{ position: 'absolute', top: '20%', left: '29%' }}>
-      {pedals.map((elem, index) => (
+    <div
+      style={{
```
Comment (thebearingedge) [inline client/components/BuilderPedals/BuilderPedals.js]: You should extract non-dynamic styles into a variable outside of your component(s).  Keep your render methods and stateless function components as short as you can.

```js
const styles = {
  pedals: {
    display: 'flex',
    flexDirection: 'row',
    // ...
  },
  icon: {
    fontSize: '40px',
    // ...
  }
}
```

```jsx
<div style={styles.pedals}/>

<Icon style={styles.icon}/>
```

File: client/components/Pedals/Pedals.js [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
-              price={elem.price}
-              type={elem.type}
-            />
-          )
-        })}
-      </Card.Group>
-    </Container>
-  )
-}
+const pedals = ({ pedalData, handleClick }) => (
```
Comment (thebearingedge) [inline client/components/Pedals/Pedals.js]: Use `UpperCase` for components.

File: client/containers/Builder/Builder.js [hunk 3]
```diff
@@ -55,14 +60,49 @@ class Builder extends Component {
...
+
+    this.setState({ pedalsOnBoard: copy })
+  }
+
+  rotateButtonHide = (id, event) => {
+    const copy = [...this.state.pedalsOnBoard]
+
+    copy.forEach((elem, index) => {
+      if (elem.id === id) {
+        elem.showRotate = false
```
Comment (thebearingedge) [inline client/containers/Builder/Builder.js]: You are mutating your object here. You need a combination of `Array.prototype.map` and `Object.assign` (or slice and spread).
https://redux.js.org/recipes/structuringreducers/immutableupdatepatterns#updating-an-item-in-an-array

File: .eslintrc [hunk 4]
```diff
@@ -10,6 +10,8 @@
     "curly": ["error", "multi-line"],
     "brace-style": ["error", "stroustrup"],
     "react/prop-types": 0,
+    "array-callback-return": 0,
```
Comment (thebearingedge) [inline .eslintrc]: Why did you disable this rule?

Comment (ArthurHwang) [reply .eslintrc]: The linter was pestering me, even after I changed it!

**Pull request merged**